log_last.log.1

```diff
@@ -0,0 +1,2 @@
+2022-03-24 21:58:25 [SEVERE] SlimeVR start-up error! Required ports are busy. Make sure there is no other instance of SlimeVR Server running.
```

Comment:
Logs in git, please add it to .gitignore. But also just don't run from the project folder to avoid the project pollution 😛...